choreolib/src/main/java/choreo/auto/AutoChooser.java

```diff
@@ -179,6 +228,25 @@ public AutoRoutine getSelectedAutoRoutine() {
     return lastAutoRoutine;
   }
 
+  /**
+   * Gets a Command that runs the latest selected auto routine,
+   * stopping when the selected routine finishes.
+   * This Command can directly be bound to a trigger, like so:
+   * <pre><code>
+   *     AutoChooser chooser = ...;
+   *
+   *     public Robot() {
+   *         RobotModeTriggers.autonomous().whileTrue(chooser.autoCmd());
+   *     }
+   * </code></pre>
+   * @return A command that runs the selected {@link AutoRoutine}
+   */
+  public Command autoCmd() {
```

Comment:
Fix up the terminology, rename this to `autoCmdScheduler`. In the docs instead of saying "run" use "schedule"